v1/topdown/bindings.go

```diff
@@ -68,7 +68,7 @@ func (u *bindings) Plug(a *ast.Term) *ast.Term {
 }
 
 func (u *bindings) PlugNamespaced(a *ast.Term, caller *bindings) *ast.Term {
-	if u != nil {
+	if u != nil && u.instr != nil {
```

Comment:
Not likely. I did this change only because it felt obvious that this was the original intention of the check, and it came with no risk (as far as I could see)